Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM php:7.0.26-fpm-jessie
+
+RUN apt-get update && apt-get install -y git-core zip
```

Comment:
Dependencies needed to run `composer`. As the comment says, there could be another way (multi-stage builds) that complicates things.